hw/lint/data/common_lint_cfg.hjson

```diff
@@ -6,22 +6,14 @@
   flow_makefile:    "{proj_root}/hw/lint/data/lint.mk"
 
   import_cfgs:      [// common server configuration for results upload
-                     "{proj_root}/hw/data/common_project_cfg.hjson"]
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/data/{tool}.hjson"]
 
   // Default directory structure for the output
   dut:              "{name}"
   build_dir:        "{scratch_path}/{build_mode}"
-
-  // We rely on fusesoc to run lint for us
-  build_cmd:  "fusesoc "
-  build_opts: ["--cores-root {proj_root}",
-               "run --target={flow} --tool={tool} --build-root={build_dir}",
-               " {fusesoc_core} "]
-  build_log: "{build_dir}/{dut}_flow.log"
-
-  report_cmd: "{proj_root}/hw/lint/tools/{tool}/parse-lint-report.py "
-  report_opts: ["--repdir={build_dir}/lint-{tool} --outdir={build_dir}"]
-  tool_srcs: []
+  build_log: "{build_dir}/lint.log"
```

Comment:
nit: please align and group like items together (build_dir and build_log).

Reply:
`build_dir` and `build_log`  are already grouped together, but the alignment of the strings on the right hand side is not correct. will fix.